.github/workflows/cypress.yaml

```
@@ -0,0 +1,109 @@
+          CYPRESS_BASE_URL: http://localhost:4321
+          REACT_APP_TALAWA_URL: http://localhost:4000/graphql
+
+  Cleanup:
+    if: always()
+    runs-on: ubuntu-latest
+    needs: [Install, Setup-Backend-Run-Tests]
+    steps:
+      - name: Remove Artifact
+        uses: geekyeggo/delete-artifact@v5
+        with:
+          name: build
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)